akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -763,8 +788,8 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
         log.error(cause, s"{} failed after shutdown. {}", streamName, cause.getMessage)
       case _: AbruptTerminationException ⇒ // ActorSystem shutdown
       case cause ⇒
-        _inboundCompressions.foreach(_.close())
-        _inboundCompressions = None
+        // inboundCompressions.foreach(_.close())
+        // _inboundCompressionAccess = None
```

Comment:
removing, some of those still laying around from when I wasn't sure if we need this or not.
With the move of compression creation to the outside I think we do need it again, so uncommenting actually :)